components/docs-chef-io/content/automate/reusable/md/restore_troubleshooting.md

```diff
@@ -19,3 +19,173 @@ Try these steps if Chef Automate returns an error while restoring data.
     ```
 
 Now check the status of the Automate node and then try running the restore command from the bastion host.
+
+1. How to change the `base_path` or `path`. Below steps for the File System backup
```

Comment:
The steps for the File System backup are as shown below:

INSTEAD OF 

Below steps for the FIle System backup